app/frontend/src/components/admin/StatusTable.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <v-card>
+    <v-toolbar flat color="grey lighten-3">
+      <v-card-title>Submission Status History</v-card-title>
+    </v-toolbar>
+
+    <v-container>
+      <v-alert v-if="showAlert" :type="alertType" tile dense>{{alertMessage}}</v-alert>
+
+      <v-data-table
+        :headers="headers"
+        :items="statuses"
+        :items-per-page="10"
+        :search="search"
+        :loading="loading"
+        loading-text="Loading... Please wait"
+        item-key="inspectionStatusId"
+        class="status-table"
+      >
+        <template v-slot:item.createdAt="{ item }">{{ formatDate(item.createdAt) }}</template>
+
+        <template v-slot:item.grade="{ item }">
+          <v-icon v-if="item.grade && item.grade === 'PASS'" color="green">check</v-icon>
+          <v-icon v-else color="red">close</v-icon>
+        </template>
+
+        <template v-slot:item.inspectorName="{ item }">{{ formatInspector(item.inspectorName) }}</template>
+
+        <template v-slot:item.inspectionDate="{ item }">{{ formatDate(item.inspectionDate) }}</template>
+      </v-data-table>
+    </v-container>
+  </v-card>
+</template>
+
+<script>
+import ipcService from '@/services/ipcService';
+
+export default {
+  name: 'SubmissionsTable',
+  computed: {
+    responsiveCell() {
+      return this.$vuetify.breakpoint.name == 'xs'
+        ? 'v-data-table__mobile-table-row'
+        : '';
+    }
+  },
+  data() {
+    return {
+      search: '',
+      headers: [
+        { text: 'Status', value: 'status' },
+        { text: 'Date', align: 'start', value: 'createdAt' },
+        { text: 'Grade', align: 'start', value: 'grade' },
+        { text: 'Assigned Inspector', value: 'inspectorName' },
+        { text: 'Inspection Date', value: 'inspectionDate' }
+      ],
+      statuses: [],
+      ipcPlanId: this.$route.params.ipcPlanId,
+      loading: true,
+      showAlert: false,
+      alertType: null,
+      alertMessage: '',
+      expanded: []
+    };
+  },
+  methods: {
+    formatDate(date) {
+      return date ? new Date(date).toLocaleString() : 'N/A';
+    },
+    formatInspector(inspectorName) {
+      return inspectorName ? inspectorName : 'N/A';
```

Comment:
blanks are probably better than N/A, could have a bunch of statuses with N/A then the real data gets lost in the clutter..., same for date.

Reply:
The date behavior was grafted over from the other table. It felt a bit jarring to see multiple rows empty, but it does also make it clear there isn't data there. I'm ok with either approach.

Reply:
Changed null representations back to blank, and added extra behavior for grade to distinguish between pass, fail and null.

Reply:
👍